src/content/guides/how-to-create-an-ethereum-account/index.md

```diff
@@ -49,20 +47,21 @@ Once you have saved your recovery phrase you should see your wallet dashboard wi
+You can use the same address on all EVM compatible blockchains. This [list](https://chainlist.org/) will show you which blockchains you can use with the same address. Some blockchains, like Bitcoin, implement a completely separate set of network rules and you will need a different address with a different format. If you have a smart contract wallet you should check its product website for more info on which blockchains are supported.
 
 ### Is having my own wallet safer than keeping my funds on an exchange?
 
-Yes, this is a much safer option because nobody else will have access to your funds. There are unfortunately many examples of failed exchanges that filed for bankruptcy resulting in users' losing their savings which were being held in custody. Hacks, frozen accounts, or blocked withdrawals are some other common risks. Owning a wallet (with a recovery phrase) is the best way to safeguard your assets. Nonetheless, a poorly backed-up recovery phrase potentially exposes you to more risks as compared to having your keys managed by an exchange. Make sure to store your recovery phrase well.
+
+Having your own wallet means you take responsibility for the security of your assets. There are unfortunately many examples of failed exchanges that lost their customers' money. Owning a wallet (with a recovery phrase) removes the risk associated with trusting some entity to hold your assets. However, you have to secure your own keys and avoid phishing scams, accidentally approving transactions or exposing keys, interacting with fake websites and other self-custody risks. The risks are different.
 
 ### If I lose my phone/hardware wallet, do I need to use the same wallet app again to recover the lost funds?
 
-No, you can use almost any wallet as the recovery process is largely standardized. This means that you can put the same 12 or 24 word phrase into most wallets and they will restore your same account. Be careful if you ever need to do this: it is best to make sure you are not connected to the internet when recovering your wallet so that your recovery phrase is not accidentally leaked. It is often impossible to recover lost funds without the recovery phrase.
+No, you can use a different wallet. As long as you have the seed phrase you can enter it into most wallets and they will restore your account. Be careful if you ever need to do this: it is best to make sure you are not connected to the internet when recovering your wallet so that your recovery phrase is not accidentally leaked. It is often impossible to recover lost funds without the recovery phrase.
```

Comment:
another term - seed phrase - needs to be unified terminology, otherwise very confusing for the reader.

Reply:
ok fair point - we should use "seed phrase" everywhere then.

Reply:
I prefer seed phrase. It does more than recover accounts - it can be used to create an account that does not already exist or recreate an accoutn across multiple devices/wallets - neither of these are "recovery"

Reply:
im still torn, the reason is i looked at onboarding process for some wallets: crypto.com, coinbase wallet and unisim wallet -> they all use "Recovery phrase" not "seed phrase". It would be good to have unified terminology with what users encounter.

Reply:
taho uses "recovery" as well, seems to me it is more widely accepted now

Reply:
@wackerow @minimalsm  whats you take here?

Reply:
I left thoughts elsewhere in this PR but I'll summarize in this thread.

Personally, I usually use both of these together to try and explain this topic, but can be tough in written format while trying to maintain brevity.

I think "seed" offers the ability to provide a better mental model around the topic. That word is used because it refers to how a whole tree of keys grows out from that initial seed. Other real-world analogies we've used on the site seem to get positive feedback from our newer users, and I think there is an opportunity for this here.

At the same time, re: SEO etc, I think it's worth adding a one-liner about the other terms that are also used, which all represent the same thing.

ie,

> You may also see this referred to as a "recovery phrase" or "mnemonic"

> You may also see this referred to as a "recovery phrase" or "mnemonic", as this phrase is easy to write down and store, and can recover all of your accounts.

...something along these lines, depending how detailed you want to be.


Not a hill I'd die on though. If we go with "recovery", then I'd use that consistently and still consider a one-liner to loop in the other terms for completeness.